scripts/dotnet/run.sh

```
@@ -88,24 +88,32 @@ if [ ! -f $src_dir/linux-arm64/libsherpa-onnx-c-api.so ]; then
 fi
 
-if [ ! -f $src_dir/android-arm64/libsherpa-onnx-c-api.so ]; then
-  echo "---android arm64---"
-  cd android-arm64
+if [ ! -f $src_dir/android-arm64/libsherpa-onnx-c-api.so ] || [ ! -f $src_dir/android-x64/libsherpa-onnx-c-api.so ]; then
+  echo "---android---"
   mkdir -p tarball
   cd tarball
-  if [ -f $android_arm64_tarball  ]; then
-    cp -v $android_arm64_tarball .
+  if [ -f $android_tarball  ]; then
```

Shellcheck (0.11.0)

[info] Double quote to prevent globbing and word splitting.

(SC2086)